config/locales/en.yml

```diff
@@ -379,15 +390,15 @@ en:
       title: Number of children living with you
     dob:
       breadcrumb: Step 14 of 20
-      hint_html: Use this format <span class="dateFormatAccessible">D D / M M / Y Y Y Y</span>
+      hint_html: For example, 01 08 2007
```

Comment:
`DD MM YYYY` may be easier to understand that the day goes first and then the month.

Reply:
@leoapost that change is to be up to date with GDS pattern, not because I just decide to change it :)